test/unit/alignment/pairwise/alignment_range_test.cpp

```diff
@@ -94,6 +122,6 @@ TEST(alignment_range, iterable)
 
 TEST(alignment_range, default_construction)
 {
-    alignment_range<dummy_executor> rng{};
-    EXPECT_TRUE(begin(rng) == end(rng));
+    seqan3::alignment_range<dummy_executor> rng{};
+    EXPECT_THROW(rng.begin(), std::runtime_error);
```

Comment:
missing `#include <stdexcept>`

Reply:
Well, that is usually not necessary. I think for tests it is ok to use indirect includes.